locales/he_IL/LC_MESSAGES/duckduckgo.po

```diff
@@ -907,14 +913,16 @@ msgstr "הקש/י על צלמית ה-%s...%s בפיונה הימנית עליו
 msgid "Click the %sGeneral%s tab."
 msgstr "לחץ על הכרטיסייה %sכללי%s"
 
+#, fuzzy
+#| msgid "Click the %s icon in the top toolbar"
 msgid "Click the %sellipsis icon%s in the toolbar."
-msgstr ""
+msgstr "לחץ/י על אייקון ה%s בסרגל הכלים העליון"
```

Comment:
It looks like here we are missing one placeholder tag. Tags should surround the words `ellipsis icon`, but I only see one placeholder tag (`%s`) in the translated string. Do you mind taking another look?

Reply:
Hello, fixed it.